templates/profiles/gui.yaml

```diff
@@ -14,11 +14,11 @@ devices:
   eth0:
     name: eth0
     nictype: bridged
-    parent: br0
+    parent: {{.Network}}
     type: nic
   PASocket:
     path: /tmp/.pulse-native
-    source: /run/user/1000/pulse/native
+    source: /run/user/{{.UID}}/pulse/native
     type: disk
   X1:
     path: /tmp/.X11-unix/X1
```

Comment:
If `{{.Display}} === 0` should this this "X1" section and the "X1" path be replaced with "X0" instead?

In #31, a couple people reported that `init` failed because "X1" did not exist. At least in one case "XO" did exist instead.

Reply:
This seems to depend on the display manager, and perhaps Wayland?  We may need more code to detect which is active.

Reply:
It may be sufficient to test for the existence of `/tmp/.X11-unix/X1` or `/tmp/.X11-unix/X0` and set up the integration for one or both if they are present.

Reply:
the gui.yaml template will need to be converted into a `text/template` that gets processed before it's installed.  That might be useful anyway for other things, like ensuring that pulse audio is installed, runtime UID's that aren't always `1000` etc.

Reply:
umm. it's already a template.  I'm going back to my coffee.

Reply:
thinking more, if both X1 and X0 exist, one is the login manager and the other is a separate X server for the login session (I *think* - this has been my personal experience).  I wonder if there's a way to determine the active X socket for the current session?

Reply:
There's got to be a way, maybe an env variable?